Documentation/Signoffs/CommunicationsSignoff.md

```diff
@@ -0,0 +1,38 @@
+# Communication Signoff
+## FUNCTION OF THE SUBSYSTEM
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; The Communication subsystem is responsible for receiving data from the Microcontroller subsystem, connecting the device to the Internet, and formatting and sending the received data to the Web & Wireless subsystem.  This subsystem will utilize 4G LTE (Long-term Evolution) as a standard in telecommunications to connect to the Internet for its near global coverage.  
+
+## CONSTRAINTS ON THE SUBSYSTEM
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;  The constraints given below are generalized to allow for comparison of any potential contender for the solution.  Constraints on the subsystem are as follows:
+1. The component must be able to utilize 4G LTE telecommunication.  4G LTE operating frequencies include 600 MHz, 700 MHz, 1700/2100 MHz, 2300 MHz, and 2500 MHz bands.  This is the chosen method of accessing the Internet due to its coverage and support in place.
+2. The component must have a slot and support for a Subscriber Identity Module card (SIM).  The component must be able to use this SIM card and its identifying information to access the specified broadband cellular network related to the SIM card's service provider.
+3. Price for this component should remain under $100 as end-users only have a budget of $300.  Allocating 1/3 of the budget is from estimating the costs of other subsystems and viewing prices of available options that satisfy the functions.
+4. For receiving data from the Microcontroller subsystem, Serial Peripheral Interface (SPI) or Universal Asynchronous Receiver/Transmitter (UART) as standards for hardware communication will need to be achievable.  
+5. There must be sufficient memory for the code. Comparing similar operating code (Wireless Sensor Collection and Small-time Processes Operating) with libraries and compilation show a range of 12 KB (Kilobyte) (10%) above or below 120 KB in flash memory. To account for worst case, 132 KB is the required memory.  
+6. The board will need to have coding language support for C and/or Python. At least one must be supported. These languages are chosen due to popularity, online support, and performance. This helps create a wider web of coverage for end-users who are at least relatively experienced with coding and engineering concepts.
+7. This project aims for replicability, so a component with a stable and reputable manufacturer that remains relevant and available is a key part.  A stable and reputable manufacturer is assumed to remain in business for years to come and continue producing the chosen board. Public and customer reviews will be used to gauge manufacturer reputability while component production history will be used to gauge product longevity.
+8. The clock speed for sending/receiving signals must be at least 4096 KHz.  If the clock were any slower, it would not able to keep up with 4G speeds.  
```

Comment:
What is the justification for this value? What are the 4G speeds?

Reply:
That value is the wrong copy-paste of the actual speed required.  On second viewing, this constraint is not necessary as the first constraint already requires that the component be able to use 4G LTE.